packages/utilities/src/store/slices/user/authApi.ts

```diff
@@ -1,92 +1,71 @@
-import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
-// import { LoginInput } from '../../pages/login.page';
-// import { RegisterInput } from '../../pages/register.page';
-import { GenericResponse, Login } from '@zuri/types';
-import { userApi } from './userApi';
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { GenericResponse, Login, RegisterInput } from "@zuri/types";
+import { userApi } from "./userApi";
+import { BASE_API_URL } from "../../../constants";
 
-const BASE_URL ='https://api.zuri.chat';
+const BASE_URL = BASE_API_URL;
 
 export const authApi = createApi({
-  reducerPath: 'authApi',
+  reducerPath: "authApi",
   baseQuery: fetchBaseQuery({
-    baseUrl: `${BASE_URL}/api/auth/`,
+    baseUrl: `${BASE_URL}/api/auth/`
   }),
-  endpoints: (builder) => ({
-    // registerUser: builder.mutation<IGenericResponse, RegisterInput>({
-    //   query(data) {
-    //     return {
-    //       url: 'register',
-    //       method: 'POST',
-    //       body: data,
-    //     };
-    //   },
-    // }),
-    loginUser: builder.mutation< { access_token: string; status: string }, Login>({
+  endpoints: builder => ({
+    registerUser: builder.mutation<GenericResponse, RegisterInput>({
       query(data) {
         return {
-          url: '/auth/login',
-          method: 'POST',
+          url: "register",
+          method: "POST",
+          body: data
+        };
+      }
+    }),
+    loginUser: builder.mutation<
+      { access_token: string; status: string },
+      Login
```

Comment:
don't use this login type here

there is no need to extract it as it will only ever be used here